libknet/handle.c

```diff
@@ -671,6 +671,7 @@ knet_handle_t knet_handle_new(knet_node_id_t host_id,
 		log_err(knet_h, KNET_SUB_HANDLE, "Unable to init handles traceker: %s",
```

Comment:
I don't know what I'm talking about, but `handles traceker` seems to be a typo in this message. Maybe `handle tracker`?

Reply:
Yep.. typo :) i´ll try to remember to schedule it in the next PR or just fine one :)